Jenkinsfile

```diff
@@ -24,36 +48,77 @@ node {
         }
     }
 
-    if (isTag()) {
-        stage('Publish') {
-            nodeEnv.inside("-e HOME=${workspace}") {
-                withCredentials([
-                    [$class: 'StringBinding', credentialsId: 'npm-token', variable: 'NPM_TOKEN']]) {
-
-                    // Use `npm` rather than `yarn` for publishing.
-                    // See https://github.com/yarnpkg/yarn/pull/3391.
-                    sh "echo '//registry.npmjs.org/:_authToken=${env.NPM_TOKEN}' >> \$HOME/.npmrc"
-                    sh "npm publish"
-                }
-            }
+    if (env.BRANCH_NAME != releaseFromBranch) {
+        echo "Skipping deployment because this is not the ${releaseFromBranch} branch"
+        return
+    }
+
+    stage('Publish') {
+        input(message: "Publish new client release?")
```

Comment:
This is what stops the deployment until the user clicks a button to continue inside Jenkins.